ci/pipeline.yml

```diff
@@ -209,6 +217,37 @@ jobs:
         GITHUB_PASSWORD: ((github-password))
         GITHUB_USERNAME: ((github-username))
       image: spring-boot-ci-image
+- name: detect-docker-updates
+  plan:
+  - get: git-repo
+    resource: git-repo-ci-docker
```

Comment:
I was wondering if that should point to the new resource.

Reply:
I don't think it should point to the new resource. We will only use the new resource to `put` to it since we want to `put` to that branch. Since the commit is based off `master`, we need to use the actual `git-repo` resource.